### DISCUSSION TITLE
Binary R Package request for Ubuntu 22 images running R 4.2

### DISCUSSION THREAD

**Discussion by JavOrraca** [duckdb/duckdb-r#1132] [Ideas]
We have several production images that run Ubuntu 22 (Jammy) w/ R 4.2.3. CRAN nor Posit Package Manager have a binary for this combo of OS and R lang 4.2.

Would it be possible to please make one available? Our Jammy and Nomad images running more recent R langs install `duckdb` in seconds via the binary vs 35 minutes from source.

If you have other ideas, I'm all ears. Thank you!

**Comment by krlmlr** [comment]
Thanks, Javier. You can get Noble binaries from r-universe. As far as I can tell, Jammy is currently out of scope. Would upgrading your images be an option?

https://duckdb.r-universe.dev/duckdb

CC @jeroen.
